modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-requirements.adoc

```diff
@@ -0,0 +1,29 @@
+= Azure System Requirements
+When setting up a SUSE Manager {payg} instance on {azure}, it's essential to consider system requirements for optimal performance and functionality. The default requirements outlined below have been tailored for smooth deployment and operation.
+
+By default, certain disks are automatically generated when establishing a {productname} {payg} instance on {azure}. To complete the setup of these disks, use the **suma-storage** tool.
+
+.Azure System Requirements
+[cols="2,1"]
+|===
+| Requirement | Details
+
+| root storage
+| 200 GB
```

Comment:
`100 GB`

Reply:
Should also be changed in the AWS specific section to `100GB` for the root partition.